Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -410,6 +410,11 @@ function PowerGenerator_JobComplete( j )
 end
 
 function LandingPad_Temp_UpdateAction(furniture, deltaTime)
+    
+    if(furniture.tile.room.ID ~= 0) then
+        return
+    end
+    
 	spawnSpot = furniture.GetSpawnSpotTile()
```

Comment:
There is a lot of tabs and space in the file, I'm not sure what we should use as a default. Is there a decision about it?